src/cpp/core/zlib/zutil.h

```diff
@@ -22,7 +22,7 @@
 #include "zlib.h"
 
 #if defined(STDC) && !defined(Z_SOLO)
-#  if !(defined(_WIN32_WCE) && defined(_MSC_VER))
+#  if (!defined(_WIN32_WCE) && defined(_MSC_VER))
```

Comment:
This also seems odd?

Reply:
@MariaSemple this feels like another accidental change since this file is not ours.
Thoughts?

Reply:
Yes, I doubt I meant to do this.